src/main/java/persistence/entity/EntityPersister.java

```diff
@@ -0,0 +1,52 @@
+package persistence.entity;
+
+import database.sql.dml.DeleteQueryBuilder;
+import database.sql.dml.InsertQueryBuilder;
+import database.sql.dml.UpdateQueryBuilder;
+import database.sql.util.EntityMetadata;
+import jdbc.JdbcTemplate;
+
+import java.util.Map;
+
+/**
+ * 엔터티의 메타데이터와 데이터베이스 매핑 정보를 제공하고,
+ * 변경된 엔터티를 데이터베이스에 동기화하는 역할
+ */
+public class EntityPersister {
+    private final JdbcTemplate jdbcTemplate;
+    private final EntityMetadata metadata;
+    private final InsertQueryBuilder insertQueryBuilder;
+    private final UpdateQueryBuilder updateQueryBuilder;
+    private final DeleteQueryBuilder deleteQueryBuilder;
+
+    public EntityPersister(JdbcTemplate jdbcTemplate, Class<?> entityClass) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.metadata = new EntityMetadata(entityClass);
+        this.insertQueryBuilder = new InsertQueryBuilder(entityClass);
```

Comment:
```suggestion
        this.insertQueryBuilder = new InsertQueryBuilder(this.metadata);
```

현재 구현하신걸보니 안에서 메타를 다시 생성하는데 그냥 메타데이터를 넣어주는것도 생각해볼수있겠네요 ㅎ
_(반영을 위한것은 아닙니다~ 나중에 어떻게 바뀔지 모르니 클래스를 넘겨줘도 될것같아요~)_